libs/extensions/angular/image-banner/src/image-banner.component.scss

```diff
@@ -71,7 +69,8 @@ $container-query-breakpoint: 600px;
 }
 
 kirby-card {
-  height: 100%;
```

Comment:
height: 100% is needed for the `slideStretchHeight` directive to function and should not be removed